src/js/component/InputList.jsx

```diff
@@ -0,0 +1,83 @@
+//Import REACT, HOOKS and PROPTYPES
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Items from "./Items.jsx";
+import todolistimage from "../../img/todolistimage.jpg";
+
+//Component
+const InputList = props => {
+	//Hooks
+	const [inputItem, setinputItem] = useState("");
+	const [list, setList] = useState([]);
+
+	//Methods
+	//Function to check and add task
+	const validateTask = () => {
+		let newObject = {
+			label: inputItem,
+			done: false
+		};
+		let newList = list.concat([newObject]);
+		setList(newList);
+	};
+
+	//Function to delete task
+	const deleteTask = item => {
+		let deleteItem = [...list];
+		deleteItem.splice(item, 1);
+		setList(deleteItem);
+	};
+	//Returns the component using the Input property and adds the list of tasks using the Map method.
```

Comment:
Please remove this unnecessary comment :)

Reply:
Deleted all unnecessary comments. :)